From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Fri, 22 Nov 2013 11:50:29 +0000
Subject: manpage hotkey

The Debian package uses "Pause" as the default hotkey, for
compatibility with the old gromit package, and also because (to be
honest) I personally use F9 for something else.  This patch document
the change in gromit-mpx(1).
---
 gromit-mpx.1 | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/gromit-mpx.1 b/gromit-mpx.1
index 49dd89a..e70c553 100644
--- a/gromit-mpx.1
+++ b/gromit-mpx.1
@@ -33,20 +33,21 @@ Gromit-MPX is activated by either a hotkey or a repeated invocation of Gromit-MP
 (the latter can e.g. used by other applications or your windowmanager).
 .br
 .SH KEYBOARD CONTROL
-By default, Gromit-MPX grabs the "F9" key (this can be changed using the
-"\-\-key" option), making it unavailable to other application. The
-available shortcuts are:
+By default, Gromit-MPX grabs the "Pause" key, making it unavailable to
+other applications.  (The use of "Pause" is Debian-specific: the
+upstream package uses "F9".  The hotkey can be changed using the
+"\-\-key" option.)  The available shortcuts are:
 .TP
-.B F9
+.B Pause
 toggle painting
 .TP
-.B SHIFT-F9
+.B SHIFT-Pause
 clear screen
 .TP
-.B CTRL-F9
+.B CTRL-Pause
 toggle visibility
 .TP
-.B ALT-F9
+.B ALT-Pause
 quit Gromit-MPX
 .PP
 .SH OPTIONS (STARTUP)
